docs/ai-functions/providers.md

```diff
@@ -196,6 +197,89 @@ This example demonstrates:
     - **Compare model outputs**: Run the same prompt through different models for quality assessment
     - **Manage rate limits**: Distribute load across multiple API keys or providers
 
+## Prompting with OpenAI-Compatible Providers
```

Comment:
See my note above - I think this section could be dramatically simplified and maybe combined with the quick provider setup